Gemfile.lock

```diff
@@ -17,56 +19,56 @@ GEM
       multipart-post (>= 1.2, < 3)
     git (1.8.1)
       rchardet (~> 1.8)
-    github_api (0.16.0)
-      addressable (~> 2.4.0)
-      descendants_tracker (~> 0.0.4)
+    github_api (0.11.3)
+      addressable (~> 2.3)
+      descendants_tracker (~> 0.0.1)
       faraday (~> 0.8, < 0.10)
-      hashie (>= 3.4)
-      mime-types (>= 1.16, < 3.0)
-      oauth2 (~> 1.0)
+      hashie (>= 1.2)
+      multi_json (>= 1.7.5, < 2.0)
+      nokogiri (~> 1.6.0)
+      oauth2
     hashie (4.1.0)
     highline (2.0.3)
-    jeweler (2.3.9)
+    jeweler (2.3.5)
       builder
-      bundler
+      bundler (>= 1.0)
       git (>= 1.2.5)
-      github_api (~> 0.16.0)
+      github_api (~> 0.11.0)
       highline (>= 1.6.15)
       nokogiri (>= 1.5.10)
-      psych
+      psych (~> 2.2)
       rake
       rdoc
       semver2
-    jwt (2.2.2)
+    jwt (2.2.3)
     method_source (1.0.0)
-    mime-types (2.99.3)
-    mini_portile2 (2.5.0)
+    mini_portile2 (2.1.0)
     multi_json (1.15.0)
     multi_xml (0.6.0)
     multipart-post (2.1.1)
-    nokogiri (1.11.2)
-      mini_portile2 (~> 2.5.0)
-      racc (~> 1.4)
+    nokogiri (1.6.8.1)
```

Comment:
== above, seems like a `nokogiri` downgrade?

Reply:
About all the `Gemfile.lock` issues. I don't know why, different bundler versions perhaps have different resolving paths.

The dependencies in this gem have issues. Gemfile.lock should not be a part of the repository, and dev dependencies should move from gemspec to Gemnfile.

I don't mind taking a stab at it as well, but thought if at all, it should be a separate PR.

Reply:
If you have time, that sounds like a very positive change.

Reply:
> If you have time, that sounds like a very positive change.

In this PR, or a separate one, after this one is merged?

Reply:
Seems like a separate PR before this one would be best, but only if you have time & agree. I'm fine with any approach :)

Reply:
a separate PR from `master` will not work on Ruby 3.x...
let me try a separate PR, but if it gets messy, we might need to merge this one first.

Reply:
I just merged https://github.com/kenpratt/wikipedia-client/pull/90, if you want to merge/rebase on that.